--- laldetchar-0.3.5/src/LALDetCharGlibTypes.h.orig	2023-01-22 20:51:02.734279401 +0100
+++ laldetchar-0.3.5/src/LALDetCharGlibTypes.h	2023-01-22 20:51:09.554242455 +0100
@@ -48,7 +48,7 @@ size_t XLALGetGSequenceLength(LALGSequen
 
 void XLALDestroyGSequenceIter(LALGSequenceIter* itr);
 #ifdef SWIG   // SWIG interface directives
-SWIGLAL(OWNED_BY_1ST_ARG(LALGSequenceIter*, XLALGSequenceBegin));
+SWIGLAL(RETURN_OWNED_BY_1ST_ARG(LALGSequenceIter*, XLALGSequenceBegin));
 #endif
 LALGSequenceIter* XLALGSequenceBegin(LALGSequence* seq);
 bool XLALGSequenceNext(LALGSequenceIter* itr);
@@ -57,7 +57,7 @@ GSequenceIter* XLALGSequenceBeginRaw(LAL
 #endif
 
 #ifdef SWIG   // SWIG interface directives
-SWIGLAL(OWNED_BY_1ST_ARG(SnglBurst*, XLALGetGSeqSnglBurst));
+SWIGLAL(RETURN_OWNED_BY_1ST_ARG(SnglBurst*, XLALGetGSeqSnglBurst));
 #endif
 SnglBurst* XLALGetGSeqSnglBurst(LALGSequenceIter* itr);
 #ifdef SWIG   // SWIG interface directives
